src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -1114,6 +1114,276 @@ public static void Contains_NullValue_ThrowsArgumentNullException()
             AssertExtensions.Throws<ArgumentNullException>("value", () => "foo".Contains(null));
         }
 
+        [Fact]
+        public static void ZeroLengthContains_StringComparison()
+        {
+            var a = new char[3];
+
+            #if !netfx
```

Comment:
out-dent these ifdefs completely to the left.

Reply:
Also, add a comment stating, this Contains overload doesn't exist on netfx.

Reply:
done